docs/source/release/v6.3.0/reflectometry.rst

```diff
@@ -15,5 +15,6 @@ Bugfixes
 --------
 - The Add Row button is now disabled in the Experiment settings tab when runs are being processed.
 - Invalid per-angle defaults values that are saved into a batch will now be marked as invalid when loaded back into the GUI.
+- Fixed un-selected runs/groups being processed after loading a batch.
```

Comment:
Should this be groups rather than runs? When I was testing the PR I found it only worked if I selected the second group. If I selected an individual run it would run on all runs.